tasks/Architectural-design.md

```diff
@@ -1,31 +1,31 @@
+ 
+Process derived from: Woolley (2009): https://www.researchgate.net/profile/Anita-Woolley/publication/220520861_Means_vs_Ends_Implications_of_Process_and_Outcome_Focus_for_Team_Adaptation_and_Performance/links/567aa32e08ae19758380fc44/Means-vs-Ends-Implications-of-Process-and-Outcome-Focus-for-Team-Adaptation-and-Performance.pdf
 
 # Stimuli
 ## The visual components
 > [to-do]
 
 ## Materials for alternative versions of the experiment 
-> If the task has multiple versions based on changing stimuli (e.g. the set of eyes in RME), if possible, provide a complete set, or a specification of the rules for generating that set. For example, in the RME task, provide all of the eye images, and a table of terms.  
-> In the case that the complete set is not possible, provide enough unique examples that the reference version of the task experiment could be completed. If there are various levels of the task, e.g., in Sudoku, include a similarly sized sample of each level.
+If the task has multiple versions based on changing stimuli (e.g. the set of eyes in RME), if possible, provide a complete set, or a specification of the rules for generating that set. For example, in the RME task, provide all of the eye images, and a table of terms.  
+In the case that the complete set is not possible, provide enough unique examples that the reference version of the task experiment could be completed. If there are various levels of the task, e.g., in Sudoku, include a similarly sized sample of each level.
 
 # Procedure
 ## Steps
-> 1. Participants were given written and video instructions of building task. 
-> 2. The participants were given scoring information. The scoring was purposefully complex to encourage trade-offs. 
-> 3. The room was video taped with the knowledge and consent of all participants
-> 4. After the video was shown, teams were given 10 minutes to work on a worksheet. The worksheet was used to decide tasks, rough-draft building structure, and a division of roles. 
-> 5. Teams turned the worksheet in to the experimenter.
-> 6. Teams were given 40 minutes to complete building task.
-> 7. At 20 minutes, the team was told 20 minutes had past.
-> 8. At 35 minutes, the teams were told 5 minutes were remaining. 
+1. Participants were given written and video instructions of building task. 
+2. The participants were given scoring information. The scoring was purposefully complex to encourage trade-offs. 
+3. The room was video taped with the knowledge and consent of all participants.
+4. After the video was shown, teams were given 10 minutes to work on a worksheet. The worksheet was used to decide tasks, rough-draft building structure, and a division of roles. 
+5. Teams turned the worksheet in to the experimenter.
+6. Teams were given 40 minutes to complete building task.
+7. At 20 minutes, the team was told 20 minutes had past.
+8. At 35 minutes, the teams were told 5 minutes were remaining.
 
 
 ## Instructions
```

Comment:
Instead of doing this verbatim, I think we can perhaps make this more concise and summarize it.

Reply:
I'd lean toward verbatim instructions when we have them. 

Ultimately we want to reproduce the underlying task if possible.